jaxrs/src/main/java/se/fortnox/reactivewizard/jaxrs/RequestLogger.java

```diff
@@ -1,28 +1,36 @@
 package se.fortnox.reactivewizard.jaxrs;
 
+import com.google.common.annotations.VisibleForTesting;
 import io.netty.handler.codec.http.HttpResponseStatus;
 import org.slf4j.Logger;
 import reactor.netty.http.server.HttpServerRequest;
 import reactor.netty.http.server.HttpServerResponse;
 
-import java.util.Collections;
+import javax.inject.Singleton;
+import java.util.HashMap;
+import java.util.Locale;
 import java.util.Map;
 import java.util.Set;
-import java.util.stream.Collectors;
+import java.util.TreeMap;
+import java.util.function.UnaryOperator;
 
-import static java.util.Collections.emptySet;
+import static java.util.Objects.requireNonNull;
+import static java.util.stream.Collectors.toMap;
 
 /**
  * Logs incoming requests to a Logger.
```

Comment:
Should we rewrite the term _incoming_ here as well? and is the request logger only used for requests handled by the server and not the client? :thinking: